slackevents/parsers.go

```diff
@@ -192,3 +192,24 @@ func ParseEvent(rawEvent json.RawMessage, opts ...Option) (EventsAPIEvent, error
 		EventsAPIInnerEvent{},
 	}, nil
 }
+
+func ParseActionEvent(payloadString string, opts ...Option) (MessageAction, error) {
+	byteString := []byte(payloadString)
+	action := MessageAction{}
+	err := json.Unmarshal(byteString, &action)
+	if err != nil {
+		return MessageAction{}, errors.New( "MessageAction unmarshalling failed")
+	}
+
+	cfg := &Config{}
+	cfg.VerificationToken = action.Token
+	for _, opt := range opts {
+		opt(cfg)
+	}
+
+	if !cfg.TokenVerified {
+		return MessageAction{}, errors.New("no")
```

Comment:
see https://github.com/nlopes/slack/pull/311#discussion_r193254321

Reply:
I was wondering why the original error from ParseEvent was just "no" lol, I changed it to the same error text.